src/views/pages/UserProfile/index.js

```diff
@@ -45,7 +48,10 @@ class UserProfilePage extends Component {
                     name: 'settings',
                     active: false
                 }
-            ]
+            ],
+            showDeleteModal: false,
+            showDeletionError: false,
+            deletionErrorShown: false
```

Comment:
Now that I think about this it might be interesting to send this as a 'confirmation' boolean for future record keeping.

Reply:
Like on the user record?